### PR TITLE
:recycle: use of driver redux-little-router

### LINKED ISSUE #136: redux-little-router driver
create a driver to have a simpliest use of the router in reaction :

```es6
 // replace
 const eventId = getRouterParam('eventId')(store)
 // by
 const eventId = router.getParam('eventId')

 // replace
 store.dispatch(push(`/organizer/event/${eventId}`))
 // by
 router.push(`/organizer/event/${eventId}`)
```

### PR DESCRIPTION
**Pull request by bpetetot**
Closes #136

### CODE REVIEW

**Review by hgwood — COMMENTED** [review]
I'm not sure that the custom selectors are now that useful. For example, is there that much value to have `router.getRouterParam('param')` instead of `router.params.param`?

**Comment by fabienjuif** [comment]
@hgwood you should access state first.
 - So you should do store.getState().router.params.param
 - Or <redux-little-routerSelector>(store.getState()) which breaks your pure function

**Comment by fabienjuif** [comment]
@bpetetot  it fixes #136  (I don't know if you forget to ad this to your first comment)

File: src/store/drivers/reduxForm.js [hunk 1]
```diff
@@ -14,12 +15,20 @@ const asyncSubmit = (name, dispatch, getState) => async (callback, ...args) => {
...
-export default ({ dispatch, getState }) => name => ({
-  getFormValues: () => getFormValues(name, state => state.form)(getState()),
-  setSubmitFailed: (...fields) => dispatch(setSubmitFailed(name, ...fields)),
-  setSubmitSucceeded: () => dispatch(setSubmitSucceeded(name)),
-  startSubmit: () => dispatch(startSubmit(name)),
-  stopSubmit: errors => dispatch(stopSubmit(name, errors)),
-  asyncSubmit: (callback, ...options) =>
-    asyncSubmit(name, dispatch, getState)(callback, ...options),
-})
+const reduxForm = (selector) => {
```
Comment (fabienjuif) [inline src/store/drivers/reduxForm.js:18]: there is a first version published (0.1.0 I believe)

Comment (bpetetot) [reply src/store/drivers/reduxForm.js:18]: No I don't think, there is no version released of the redux-form driver
It's still pending in the PR https://github.com/alakarteio/k-ramel/pull/115

**Comment by hgwood** [comment]
Right, sorry I got confused.

File: src/store/drivers/reduxForm.js (hunk 1 above)
Comment (fabienjuif) [reply src/store/drivers/reduxForm.js:18]: You have to check npm for published packages 😀
https://www.npmjs.com/package/@k-ramel/driver-redux-form

I was not ok to merge it right now but after arguing and for our current client, we published a first version of it (0.1.0) 😛

Comment (bpetetot) [reply src/store/drivers/reduxForm.js:18]: ok, I miss that, because I didn't see it in the master branch

Comment (fabienjuif) [reply src/store/drivers/reduxForm.js:18]: Yeah I know, that was the purpose of my comment, let you know about this ;)

File: src/screens/speaker/inviteSpeaker/inviteSpeaker.container.js [hunk 2]
```diff
@@ -18,9 +16,9 @@ const mapStore = (store) => {
     uidInvite,
     join: () => {
       store.dispatch({ type: '@@ui/ADD_SPEAKER_TO_TALK', payload: { uid, talkId } })
-      store.dispatch(push(`/speaker/talk/${talkId}`))
+      router.push(`/speaker/talk/${talkId}`)
     },
-    cancel: () => store.dispatch(push('/speaker')),
+    cancel: () => router.push('/speaker'),
```
Comment (fabienjuif) [inline src/screens/speaker/inviteSpeaker/inviteSpeaker.container.js:21]: I really have the feeling this is best to decorelate UI actions from reactions (mutate store).

`cancel: () => dispatch('@@ui/INVITE_SPEAKER>ON_CANCEL')`

`when('@@ui/INVITE_SPEAKER>ON_CANCEL')((action, store, { router }) => router.push('/speaker'))`

Because you can follow (this is still hard to follow, but you can), why the router change in redux-devtools.
But I understand this a little gain for a lot more code

**Review by fabienjuif — APPROVED** [review]
👌 nice work
